Paintroid/src/androidTest/java/org/catrobat/paintroid/test/espresso/LayerIntegrationTest.kt

```diff
@@ -84,569 +98,569 @@ class LayerIntegrationTest {
-                .checkLayerCount(2)
+            .checkLayerCount(2)
         TopBarViewInteraction.onTopBarView()
-                .performRedo()
+            .performRedo()
         LayerMenuViewInteraction.onLayerMenuView()
-                .checkLayerCount(1)
+            .checkLayerCount(1)
     }
 
     @Ignore("Fail due to Pipette")
     @Test
     fun testLayerOrderUndoDelete() {
         ToolBarViewInteraction.onToolBarView()
-                .performSelectTool(ToolType.FILL)
+            .performSelectTool(ToolType.FILL)
         DrawingSurfaceInteraction.onDrawingSurfaceView()
-                .perform(UiInteractions.touchAt(DrawingSurfaceLocationProvider.MIDDLE))
+            .perform(UiInteractions.touchAt(DrawingSurfaceLocationProvider.MIDDLE))
         LayerMenuViewInteraction.onLayerMenuView()
-                .performOpen()
-                .performAddLayer()
-                .checkLayerCount(2)
-                .performClose()
+            .performOpen()
+            .performAddLayer()
+            .checkLayerCount(2)
+            .performClose()
         ToolPropertiesInteraction.onToolProperties()
-                .setColorResource(R.color.pocketpaint_color_picker_green1)
-                .checkMatchesColorResource(R.color.pocketpaint_color_picker_green1)
+            .setColorResource(R.color.pocketpaint_color_picker_green1)
+            .checkMatchesColorResource(R.color.pocketpaint_color_picker_green1)
         DrawingSurfaceInteraction.onDrawingSurfaceView()
-                .perform(UiInteractions.touchAt(DrawingSurfaceLocationProvider.MIDDLE))
+            .perform(UiInteractions.touchAt(DrawingSurfaceLocationProvider.MIDDLE))
         LayerMenuViewInteraction.onLayerMenuView()
-                .performOpen()
-                .performSelectLayer(1)
-                .performDeleteLayer()
-                .performClose()
-                .checkLayerCount(1)
+            .performOpen()
+            .performSelectLayer(1)
+            .performDeleteLayer()
+            .performClose()
+            .checkLayerCount(1)
         TopBarViewInteraction.onTopBarView()
-                .performUndo()
+            .performUndo()
         LayerMenuViewInteraction.onLayerMenuView()
-                .checkLayerCount(2)
+            .checkLayerCount(2)
         ToolBarViewInteraction.onToolBarView()
-                .performSelectTool(ToolType.PIPETTE)
+            .performSelectTool(ToolType.PIPETTE)
         DrawingSurfaceInteraction.onDrawingSurfaceView()
-                .perform(UiInteractions.touchAt(DrawingSurfaceLocationProvider.MIDDLE))
+            .perform(UiInteractions.touchAt(DrawingSurfaceLocationProvider.MIDDLE))
         ToolPropertiesInteraction.onToolProperties()
-                .checkMatchesColorResource(R.color.pocketpaint_color_picker_green1)
+            .checkMatchesColorResource(R.color.pocketpaint_color_picker_green1)
     }
 
     @Test
     fun testUndoRedoLayerRotate() {
```

Comment:
This test fails locally too, can you take a look at this one? I dont see the error on the first look, but it does fail consistently on all runs of this PR

Reply:
hmm... ok I'll take a look at it.

Reply:
<img width="602" alt="image" src="https://user-images.githubusercontent.com/52480794/123501394-33da0f80-d662-11eb-86d3-9971369d466c.png">
all the tests pass locally, I think these 2 failing tests are flaky as they fail in most of the PRs🤔, and also I didn't make any change to the logics of any code in this PR😅

Reply:
You are right. this testfile seems to be flaky, it failed the first 3 times i tried it but then worked the next few times